core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -95,7 +95,7 @@ export class QuestionBackendApiService {
    * Does basic validation on input.
    */
   private validateRequestParameters(
-      skillIds, questionCount: any,
+      skillIds: any, questionCount: any,
```

Comment:
Can you add the types here? skillIds is a list of strings and questionCount is a number (based on validation in L100 and L105 below).

Reply:
Done

Reply:
Not done?